apps/nextjs/src/components/svg/icons.tsx

```
@@ -0,0 +1,198 @@
+      width="1em"
+      height="1em"
+      viewBox="0 0 24 24"
+      {...props}
+    >
+      {" "}
+      <g fill="none" stroke="currentColor" strokeWidth={1.5}>
+        {" "}
+        <circle cx={12} cy={6} r={4}>
+          {" "}
+        </circle>
+        <path d="M20 17.5c0 2.485 0 4.5-8 4.5s-8-2.015-8-4.5S7.582 13 12 13s8 2.015 8 4.5Z"></path>{" "}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)